drake/examples/HolonomicDrive/oneWheelGradients.m

```diff
@@ -0,0 +1,43 @@
+% Autogenerated file, from makeGrads.m
```

Comment:
Not including the generated file would introduce a dependency on the Symbolic Math Toolbox, no?